src/content/apis/weather/weatherAPI.md

```diff
@@ -0,0 +1,10 @@
+---
+name: Weather API
+description: It is a powerful fully managed free weather and geolocation API provider that provides extensive APIs that range from the weather forecast, historical weather, future weather, weather alerts, air quality data, IP lookup, and astronomy through to sports, time zone, and geolocation.
+categories:
+  - weather
+features:
+  https: true
+url: https://www.weatherapi.com/docs/
+logo: https://cdn.weatherapi.com/v4/images/weatherapi_logo.png

```

Comment:
The logo has to be downloaded and place inside the repository itself.
Please check the contributing guides for more details.